src/content/en/2019/performance.md

```diff
@@ -0,0 +1,266 @@
+---
+part_number: II
+chapter_number: 7
+title: Performance
+authors: [rviscomi]
+reviewers: [JMPerez, obto, sergeychernyshev, zeman]
+---
+
+## Intro
+
+Performance is a visceral part of the user experience. For many websites, an improvement to the user experience by speeding up the page load time aligns with an improvement to conversion rates. Conversely, when performance is poor, users don't convert as often and have even been observed [rage clicking](https://blog.fullstory.com/rage-clicks-turn-analytics-into-actionable-insights/) on the page in frustration. For example, [WPO Stats](https://wpostats.com/) is a large collection of case studies showing the correlation between performance and business metrics.
```

Comment:
I don't think the phrase "For example" fits here. What about something like: "As case be seen in" or "Many companies have published case studies showing this correlation and can be seen here"

Reply:
Agreed. Since WPO Stats is more of a reference than a discussion point I'll just delete the sentence and add the link elsewhere.